spec/i18n_spec.rb

```diff
@@ -6,14 +6,20 @@
   let(:unused_keys) { i18n.unused_keys }
   let(:inconsistent_interpolations) { i18n.inconsistent_interpolations }
 
+  def details(keys)
+    keys.leaves.map do |node|
+      { key: node.full_key, file: node.data[:path] }
```

Comment:
`node.inspect` would also be a reasonable choice here. I think missing key name and file path are minimally sufficient to find and fix the issue.